sdk/src/integrationTest/java/MirrorNodeCloseClientIntegrationTest.java

```diff
@@ -0,0 +1,51 @@
+import com.hedera.hashgraph.sdk.*;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.threeten.bp.Instant;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Objects;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class MirrorNodeCloseClientIntegrationTest {
+    @Test
+    @DisplayName("Can close the Client while subscribed to mirror node topic queries")
+    void canCloseClient() throws Exception {
+        var testEnv = new IntegrationTestEnv(1);
+
+        var response = new TopicCreateTransaction()
+            .setAdminKey(testEnv.operatorKey)
+            .setTopicMemo("[e2e::TopicCreateTransaction]")
+            .execute(testEnv.client);
+
+        var topicId = Objects.requireNonNull(response.getReceipt(testEnv.client).topicId);
+
+        new TopicInfoQuery()
+            .setTopicId(topicId)
+            .execute(testEnv.client);
+
+        Thread.sleep(3000);
```

Comment:
Thread sleeps are always a code smell and should be removed. Internally, the TopicInfoQuery and TopicMessageQuery should already retry the request. If need be, that retry mechanism could be adjusted instead of sleeping.